src/components/creditNote/CreditNoteFormCalculation.tsx

```diff
@@ -176,8 +176,6 @@ export const CreditNoteFormCalculation = ({
         : (estimationData?.creditNoteEstimate?.maxCreditableAmountCents || 0) >
           (estimationData?.creditNoteEstimate?.maxRefundableAmountCents || 0),
     }
-    // IMPORTANT: not not add feeForEstimate to the dependencies, as it will cause an unexpected pre-reload of the prompted data, before BE has time to respond
```

Comment:
Why removing the important comment? That has a typo BTW 😓

Reply:
Because adding `feeForEstimate` to the depedencies now triggers an error in the linter